config/path.go

```diff
@@ -66,3 +101,20 @@ func absolutePath(value string) string {
 	clog.Errorf("cannot determine absolute path for '%s'", value)
 	return value
 }
+
+// resolveGlob evaluates glob expressions in a slice of paths and returns a resolved slice
+func resolveGlob(sources []string) []string {
+	resolved := make([]string, 0, len(sources))
+	for _, source := range sources {
+		if strings.ContainsAny(source, "?*") {
```

Comment:
I think matching characters should be "?\*[" since you can use expressions like [aA]*

Reply:
Thanks a lot for the review. Not using this very often with glob so I overlooked it. Tests are updated as well.